src/components/AdReportModal.jsx

```diff
@@ -48,7 +48,7 @@ const AdReportModal = ({ isChannelChanged }) => {
       title="현재 방송이 광고인지 선택해주세요."
       subTitle="광고일 경우, 채널을 변경할 수 있습니다."
     >
-      <div className={`flex w-[280px] flex-col items-start ${baseMinHeight}`}>
+      <div className={`flex w-[320px] flex-col items-start ${baseMinHeight}`}>
```

Comment:
```suggestion
      <div className={`flex w-80 flex-col items-start ${baseMinHeight}`}>
```
tailwindcss에서 제공하는 클래스를 사용하면 더 좋을 것 같습니다!

Reply:
좋은 포인트 감사합니다! 해당 부분은 w-80으로 변경해서 Tailwind에 맞췄습니다👍